akka-javasdk/src/main/scala/akka/javasdk/impl/agent/AgentImpl.scala

```diff
@@ -228,7 +223,7 @@ private[impl] final class AgentImpl[A <: Agent](
     val responseMessages: Seq[SessionMessage] =
       responses.map {
         case res: SpiAgent.ModelResponse =>
-          val requests = res.toolRequests.map { req =>
```

Comment:
Any strong reason to have renamed this? 

`toolCalls` is too generic. We have the requests and the responses. That's why the types an fields are being suffixed with `Request` and `Response`.